modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -116,3 +116,42 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetches autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return array Autoloaded data as option names and their sizes.
+ */
+function perflab_aao_query_autoloaded_options() {
+	global $wpdb;
+
+	$option_threshold = apply_filters( 'perflab_aao_autoloaded_options_table_threshold', 100 );
```

Comment:
@mukeshpanchal27 As discussed in Slack, I think the proposed description could be a little clearer as to what the threshold refers too.

```suggestion
	/**
	 * Filters the threshold for an autoloaded option to be considered large.
	 *
	 * The Site Health report will show users a notice if any of their autoloaded
	 * options exceed the threshold for being considered large. This filters the value
	 * for what is considered a large option.
	 *
	 * @since n.e.x.t
	 *
	 * @param int $option_threshold Threshold for an option's value to be considered
	 *                              large, in bytes. Default 100.
	 */
	$option_threshold = apply_filters( 'perflab_aao_autoloaded_options_table_threshold', 100 );
```

Reply:
Thanks @peterwilsoncc for the quick suggestion. Look good to me.